storage/tianmu/vc/expr_column.cpp

```diff
@@ -122,6 +122,12 @@ int64_t ExpressionColumn::GetValueInt64Impl(const core::MIIterator &mit) {
   return last_val_->Get64();
 }
 
+int64_t ExpressionColumn::GetValueInt64Direct(const core::MIIterator &mit) {
```

Comment:
can we use ExpressionColumn::GetValueInt64Impl() to get the value from dummy table?

Reply:
please refer to: https://github.com/stoneatom/stonedb/pull/1247#issuecomment-1427520408
the function is not used to get dummy table value;  and this modification is  SP( user defined function）thas has dml
sql， 

check function TempTable::SendResult(int64_t limit, int64_t offset, ResultSender &sender, bool pagewise) as below，
SP is evaluated once （insert one row）in LINE474, SP is evaluated the second time（insert one row again) in LINE477 (GetValueInt64), 

GetValueInt64Direct means just get value without evaluating;

The twice insetted row is trigged by this PR, it has the issue even without the PR;
 ```
LINE470:  if (ct == common::ColumnType::INT || ct == common::ColumnType::MEDIUMINT || ct == common::ColumnType::SMALLINT ||
LINE471:      ct == common::ColumnType::BYTEINT || ct == common::ColumnType::NUM || ct == common::ColumnType::BIGINT ||
LINE472:      ct == common::ColumnType::BIT) {
LINE473:    auto data_ptr = new types::TianmuNum();
LINE474:    if (vc->IsNull(it))
LINE475:      data_ptr->SetToNull();
LINE476:    else
LINE477:      data_ptr->Assign(vc->GetValueInt64(it), col->Type().GetScale(), false, ct);
LINE478:    record.emplace_back(data_ptr);
```

Reply:
please add some comments for these new added functions, to expain why they are added.